rasa/server.py

```diff
@@ -1292,7 +1292,9 @@ async def parse(request: Request) -> HTTPResponse:
             data = emulator.normalise_request_json(request.json)
             try:
                 parsed_data = await app.agent.parse_message_using_nlu_interpreter(
-                    data.get("text")
+                    data.get("text"),
+                    None,
+                    data.get("metadata")
```

Comment:
```suggestion
                    metadata=data.get("metadata")
```

Reply:
could you please add a test for this to avoid regression?

Reply:
@shaswat-indian

Reply:
Sure, will add a test. Can you help me with where exactly to check in the test code? I'm unable to find any reference doc for test files in the contributing guide.

Reply:
There are related tests [here](https://github.com/RasaHQ/rasa/blob/cfe686f67a7336b4cbccd6fee83531043cc9289e/tests/test_server.py#L376). I think a good test would be to have small NLU model which includes Duckling. We could then mock the API and inspect whether the HTTP request to Duckling includes the timezone? Or a bit simpler : Mock the `process` method the `Duckling` component to see whether this data was passed in